Plot/Plot.gradle

```diff
@@ -20,6 +20,8 @@ dependencies {
 
     compileOnly 'javax.inject:javax.inject:1'
 
+    implementation 'com.google.guava:guava:19.0'
```

Comment:
I know this was depended on implicitly before, but I'm wondering if we want to try and remove our dependency on guava, or update to a more recent version? (19.0 from 2015)

https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-8908
https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2018-10237

At the very least, we should have a follow-up ticket.

Reply:
Yep, this was picked to ensure we don't change our current version. 

We are not susceptible to either cve, I actually checked this while waiting for a build - we don't use the classes required by either, and for CVE-2018-10237 we don't allow guava objects to be serialized and sent to us at all at this time (w2w might support this in theory some day, but java serialization is already a minefield, and we would want to only enable that feature for trusted other servers).